akka-multi-node-testkit/src/main/scala/akka/remote/testkit/MultiNodeSpec.scala

```diff
@@ -160,29 +160,43 @@ object MultiNodeSpec {
   require(selfName != "", "multinode.host must not be empty")
 
   /**
-   * Port number of this node. Defaults to 0 which means a random port.
+   * TCP Port number to be used when running tests on TCP. 0 means a random port.
    *
    * {{{
-   * -Dmultinode.port=0
+   * -Dmultinode.tcp.port=0
    * }}}
    */
-  val selfPort: Int = Integer.getInteger("multinode.port", 0)
+  val tcpPort: Int = Integer.getInteger("multinode.tpc.port", 0)
```

Comment:
MultiNodeSpec is public api and even though we can break binary compatibility in testkits we probably want to be backwards compatibility in the behavior instead of silently changing from `-Dmultinode.port` to `-Dmultinode.tcp.port`

Reply:
maybe we can look for `multinode.port` first and then use `multinode.tpc.port`?

Reply:
I haven't see this comment, so late reaction. 

The `selfPort` is back and by default will be the same as the `tcpPort`, unless we explicitly pass `multinode.protocol = udp` in which case it will point to the `udpPort`